### ISSUE TITLE
[Feature Request] Open source Vector Graphics code

### ISSUE THREAD

**Issue by sbeca** [Unity-Technologies/vector-graphics-samples#3]
Please make the repository holding the com.unity.vectorgraphics package public.

We use our own internal SVG library for many of our projects and we're very interested in transitioning to a Unity provided library if possible. In saying that, we've had to make many changes and improvements to our code over time so not having access to source code would be a big negative to us.

**Comment by Frozenfire92** [comment]
I agree, I think there are a lot of people in the community who would love to contribute as this is a huge feature for 2D

**Comment by siriusye** [comment]
Is the vectorgraphics package opensource now?

**Comment by josefgrunig** [comment]
Totally agree, there is bug in the library and we could help nailing it down with the source code available!
